documents/final-submission-document/chapters/2_analysis.tex

```diff
@@ -73,11 +73,25 @@ \subsection{IETF Internet-Draft: Using XMPP for Security Information Exchange}\l
 
 The draft describes a trust model, thread model as well as specific countermeasures, e.g. to use at least TLS 1.2. These countermeasures also define restrictions of the XMPP protocol and its extensions, e.g. by limiting the topic access models of \gls{pubsub} to whitelist and authorized only~\cite{ietf-mile-xmpp-grid-05}.
 
-
 \section{Requirements Analysis}
 % NFR, priorisierung, Testbarkeit, Accessibility
 % Anforderungen abgenommen?
 
+We collected the functional requirements in the form of user stories.
+User Stories are an established and widespread concept for describing and managing requirements in agile software projects.
+In comparison to traditional tools for requirement analysis, user stories are more blured, levaing more space for changes. \cite{wirdemann2017scrum}
+
+We created some separate user stories for non-functional requirements.
+Additional non-functional requirements can be added during the project in the form of Constraints. \cite{wirdemann2017scrum}
```

Comment:
I just noticed that I always make the citations before the dot, and you always after it. On which do we settle? 😄

Reply:
In RKI, we learned that adding the citation in the same sentence marks this sentence as "cited" whereas adding it after the period (at the end of a paragraph) means that the whole paragraph is based on statements of the cited source. I try to follow that rule although I might have made a mistake here and there ...